src/components/App.js

```diff
@@ -63,7 +64,7 @@ function App() {
         await Font.loadAsync(customFonts);
         // Artificially delay for two seconds to simulate a slow loading
         // experience. Please remove this if you copy and paste the code!
-        await new Promise((resolve) => { setTimeout(resolve, 2000); });
+        // await new Promise((resolve) => { setTimeout(resolve, 2000); });
```

Comment:
Actually, I was liking the first loader saying "Launching the app" ^^ but as the comment says, we can disable it.

Reply:
Actually, I forgot to comment back this line. I just commented it for faster testing, should I add it back ?

Reply:
Yes please. :-)